#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-make.dpatch by David Kubicek <dave@awk.cz>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change installation paths in Makefile

@DPATCH@
diff -urNad cntlm~/Makefile cntlm/Makefile
--- cntlm~/Makefile	2007-06-01 19:01:11.000000000 +0200
+++ cntlm/Makefile	2007-06-01 19:16:12.000000000 +0200
@@ -2,9 +2,9 @@
 # You can tweak these three variables to make things install where you
 # like, but do not touch more unless you know what you are doing. ;)
 #
-SYSCONFDIR=/usr/local/etc
-BINDIR=/usr/local/bin
-MANDIR=/usr/local/man
+SYSCONFDIR=$(DESTDIR)/etc
+BINDIR=$(DESTDIR)/usr/sbin
+MANDIR=$(DESTDIR)/usr/share/man
 
 #
 # Careful now...
